components/userRequest.tsx

```diff
@@ -0,0 +1,12 @@
+// eslint-disable-next-line import/no-duplicates
+import { PrismaClient } from "@prisma/client";
+// eslint-disable-next-line import/no-duplicates
+import { User } from "@prisma/client";
```

Comment:
you can combine lines 2 and 4! 
`import { PrismaClient, User } from "@prisma/client";`